index.html

```diff
@@ -61,6 +61,324 @@ <h1 class="display-5 fw-bold primary-text">I’m Yannick<br> Glad to see you!</h
       </div>
   </section>
 
+  <section class="work-section section-pd-50-50 section-mg-50-0">
+
+     <div class="container">
+
+        <div class="flex-container">
+          <div class="flex-item-left">
+              
```

Comment:
- [ ] Kindly wrap the entire works segment in a `section` tag and set it to display `grid`.

Reply:
Can you explain me this part what do you mean ? I didn't get you well

Reply:
Something like this:
```
<ul class='work-section-wrap'>
  <li class='work-section'>
     <div class='container'>
     ...
     </div>
  </li>
  <li class='work-section'>
    <div class='container'>
     ...
     </div>
  </li>
</ul>
```